src/chocolatey/GetChocolatey.cs

```diff
@@ -500,7 +502,9 @@ private void extract_resources()
 
             try
             {
+#if !LibOnly
                 AssemblyFileExtractor.extract_all_resources_to_relative_directory(_container.GetInstance<IFileSystem>(), Assembly.GetAssembly(typeof(ChocolateyResourcesAssembly)), ApplicationParameters.InstallLocation, folders, ApplicationParameters.ChocolateyFileResources);
+#endif

```

Comment:
Would it be better to surround the entire try/catch with `#if !LibOnly`? I'm not sure how it likes an empty try with a filled out catch.